packages/components/src/components/form/Select/Select.tsx

```diff
@@ -170,7 +170,7 @@ const Wrapper = styled.div<WrapperProps>`
         flex-wrap: nowrap;
         min-width: ${({ $minValueWidth }) => $minValueWidth};
         justify-content: ${({ $isClean }) => ($isClean ? 'flex-end' : 'flex-start')};
-        padding: 0;
+        padding: 0 25px 0 0;
```

Comment:
We try to use predefined spacings if possible.

Could you use it this way? 🙏 
`padding: 0 ${spacingsPx.xl} 0 0;`